data-otservbr-global/scripts/globalevents/vip/online_coins.lua

```diff
@@ -31,7 +31,7 @@ function onlineCoinsEvent.onThink(interval)
 
 	local checkIp = {}
 	for _, player in pairs(players) do
-		if player:getAccountType() >= ACCOUNT_TYPE_GAMEMASTER then
+		if player:getGroup():getId() >= GROUP_TYPE_SENIORTUTOR then
```

Comment:
Creio que seja só ">" já que o senior tutor é apenas um jogador comum.
```suggestion
		if player:getGroup():getId() > GROUP_TYPE_SENIORTUTOR then
```